app/views/publish/providers/study_sites/index.html.erb

```diff
@@ -16,10 +16,16 @@
     <% end %>
 
     <% if @study_sites.none? %>
-      <p class="govuk-body">A study site, such as a university campus, is where trainees do theoretical training. Add study sites for your organisation and then link one or more to a course from the ‘Basic details’ tab on the course page.</p>
+      <p class="govuk-body">A study site, such as a university campus, is where trainees do theoretical training. Add study sites for your organisation and then attach one or more to a course from the ‘Basic details’ tab on the course page.</p>
```

Comment:
This appears in both branches. Couldn't it be moved out of the if block?

Reply:
Whoops 🙈 

https://github.com/DFE-Digital/publish-teacher-training/pull/3799/commits/d094461bcf76ad2203edde2a950b3c9795e0fd0b